cves/CVE-2012-2775.yml

```diff
@@ -308,8 +328,11 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      This vulnerability is contingent on trusting that the error checking
+      values in the file align with the actual content. If it was malicious,
+      it could overwrite into memory it shouldn't have acces to.
```

Comment:
Ironically it was part of the error checking functionality - so it was part of a feature that was designed to not trust inputs. Not disagreeing with you just observing if you want to add that.

Reply:
I think I made a mistake with respect to the PARCOR coefficients, in that they aren't actually part of error checking. Rather, they define part of the shape of the audio wave itself - something to do with differentials. It's tremendously complicated. I initially made the connection to parity bits, but they're actually not redundant information. However, like parity bits, they're still supposed to conform to the _rest of the audio file_, which is what they didn't check for.